### ISSUE TITLE
Accept Submission: Select Files -> Empty list of files

### ISSUE THREAD

**Issue by rmsoler** [pkp/pkp-lib#10102]
**Describe the bug**
When a submission is accepted an empty file list appears in step3 "Select files"

**To Reproduce**
1. Accept submission
2. Go to Step 3 "Select files"
3. There are no files to select, the list of files is empty.

**What application are you using?**
OJS 3.4

**Additional information**
Please see the screenshot below for details.
![Ojs_Error_Screen5](https://github.com/pkp/pkp-lib/assets/116004770/18b2169c-3cd2-42da-9bbe-f97923e8221f)
![Ojs_Error_Screen4](https://github.com/pkp/pkp-lib/assets/116004770/c8d264ac-8a15-44d3-a263-29194dc071b6)
![Ojs_Error_Screen3](https://github.com/pkp/pkp-lib/assets/116004770/febf4551-2ac1-4836-86cf-d9a057853261)
![Ojs_Error_Screen2](https://github.com/pkp/pkp-lib/assets/116004770/0acbfdac-a869-4367-aa07-8bc53114a045)
![Ojs_Error_Screen1](https://github.com/pkp/pkp-lib/assets/116004770/fca18654-97b1-4edb-8d88-716cbcca8f20)

**Comment by asmecher** [comment]
@rmsoler, the files that are available for selection in Step 3 of the Accept workflow are taken from the Revisions list, which is empty in at least one of your screenshots:
![image](https://github.com/pkp/pkp-lib/assets/200411/4149fc67-a1bb-48c0-9681-02220b04679b)

If you want to pull forward additional files, you can always do that from the Copyediting workflow using the Upload/Select Files button:
![image](https://github.com/pkp/pkp-lib/assets/200411/ce8aed40-513c-464e-96d7-eca5d6361c6a)
Then, check the checkbox to gain access to files in all previous stages:
![image](https://github.com/pkp/pkp-lib/assets/200411/589f1c0b-5601-42cb-9b54-9273196bf984)

Next time, the best place to start is the support forum (https://forum.pkp.sfu.ca/); then if an issue turns out to be a bug, we can open an entry here.

Long story short, I think this is intentional behaviour, though feel free to propose a change in the Feature Requests area of the forum if you have something specific in mind.

**Comment by rmsoler** [comment]
Thanks for the quick response, but I think this is a BUG because the Revisions list of the Round 1 is not empty as you can see in the screenshot.

In another screenshot you can see, in Round 2, that the file uploaded for review has been accepted, the status is "Accept Submission" state, so this file should be available because this file has ben accepted without changes by the revisor.

In the Step 1 and Step2,  these files are available to include in the emails clicking on Attach Files, so why are they not available in the last step if they are the final files?

And finally, in the OJS 3.3 version, when a user accept a submissions, these files could be selected.

We will inform our client that this is the correct operation of OJS 3.4, but it is a operation that reduces capabilities with respect to OJS 3.3 and also does not seem appropriate for a good publication workflow.

Thank you.

**Comment by mpbraendle** [comment]
I think the dialog is misleading - if there are no files to select for the copyediting stage  one shouldn't display "Select files" in step 3. Our editors were, at least, confused.

**Comment by rmartinezsoler** [comment]
Yes, agree, but also if a file has been approved by the reviewers should be available for select in this step. 
In the prevous version of OJS this work fine, the reviewed files are available for selection.

**Comment by asmecher** [comment]
@Vitaliy-1, could you have a look at this and a think?
